### ISSUE TITLE
Add https: support to cpancover.com

### ISSUE THREAD

**Issue by jsoref** [pjcj/Devel--Cover#340]
Please add `https:` support to cpancover.com. 
Generally, it's easy to maintain a certificate, e.g. using https://certbot.eff.org/

The performance for https hasn't been a significant problem in ages: https://www.keycdn.com/blog/https-performance-overhead

**Comment by pjcj** [comment]
Yes, this is definitely on my list.  There are a few more pressing problems with the server but I will get to this.

**Comment by pjcj** [comment]
The new server supports https now.  Thanks for the reminder.
